clients/dos/src/main/kotlin/DosModel.kt

```diff
@@ -90,8 +99,8 @@ data class JobRequestBody(
     /** The key of the previously uploaded ZIP file to scan. */
     val zipFileKey: String,
 
-    /** The list of purls whose packages' source code is contained in the ZIP file. */
-    val purls: List<String>
+    /** The list of purls and declared licenses for the packages whose source code is contained in the ZIP file. */
```

Comment:
This could now say just: "The list of packages whose source code is contained in the ZIP file."